tests/integration/test_view_for_github_webhooks.py

```diff
@@ -202,6 +204,117 @@
+            "email": "trzcinska.ania@gmail.com",
+            "login": "owner",
+            "id": 1,
+            "node_id": "MDQ6VXNlcjIxODIwNzg4",
+            "avatar_url": "https://avatars3.githubusercontent.com/u/21820788?v=4",
+            "gravatar_id": "",
+            "type": "User",
+            "site_admin": False
+        },
+        "html_url": "https://github.com/Codertocat/Hello-World",
+        "description": None,
+        "fork": False,
+        "url": "https://github.com/Codertocat/Hello-World",
+        "teams_url": "https://api.github.com/repos/Codertocat/Hello-World/teams",
+        "hooks_url": "https://api.github.com/repos/Codertocat/Hello-World/hooks",
+        "issue_events_url": "https://api.github.com/repos/Codertocat/Hello-World/issues/events{/number}",
+        "events_url": "https://api.github.com/repos/Codertocat/Hello-World/events",
+        "assignees_url": "https://api.github.com/repos/Codertocat/Hello-World/assignees{/user}",
+        "branches_url": "https://api.github.com/repos/Codertocat/Hello-World/branches{/branch}",
+        "tags_url": "https://api.github.com/repos/Codertocat/Hello-World/tags",
+        "blobs_url": "https://api.github.com/repos/Codertocat/Hello-World/git/blobs{/sha}",
+        "git_tags_url": "https://api.github.com/repos/Codertocat/Hello-World/git/tags{/sha}",
+        "git_refs_url": "https://api.github.com/repos/Codertocat/Hello-World/git/refs{/sha}",
+        "commits_url": "https://api.github.com/repos/Codertocat/Hello-World/commits{/sha}",
+        "git_commits_url": "https://api.github.com/repos/Codertocat/Hello-World/git/commits{/sha}",
+        "comments_url": "https://api.github.com/repos/Codertocat/Hello-World/comments{/number}",
+        "created_at": 1557933565,
+        "updated_at": "2019-05-15T15:20:41Z",
+        "pushed_at": 1557933657,
+        "git_url": "git://github.com/Codertocat/Hello-World.git",
+        "ssh_url": "git@github.com:Codertocat/Hello-World.git",
+        "clone_url": "https://github.com/Codertocat/Hello-World.git",
+        "svn_url": "https://github.com/Codertocat/Hello-World",
+        "homepage": None,
+        "size": 0,
+        "stargazers_count": 0,
+        "watchers_count": 0,
+        "language": "Ruby",
+        "disabled": False,
+        "open_issues_count": 2,
+        "license": None,
+        "forks": 1,
+        "open_issues": 2,
+        "watchers": 0,
+        "default_branch": "master",
+        "stargazers": 0,
+        "master_branch": "master"
+    },
+    "pusher": {
+        "name": "owner",
+        "email": "trzcinska.ania@gmail.com"
+    },
+    "sender": {
+        "login": "owner",
+        "id": 1,
+        "node_id": "MDQ6VXNlcjIxODIwNzg4",
+        "gravatar_id": "",
+        "type": "User",
+        "site_admin": False
+    }
+}
+
+snapshot_payload_no_commits = {
+    'link': None,
+    'branch': 'mybranch',
+    'commit': [],
+    'event_type': 'push',
+    'author': {'name': 'owner', 'id': 1}
+}
```

Comment:
when no commits means push event was on tag, so should throw an error, so we need test case that raised 400, if you don't know how to write that, you can leave it, and just create an issue for later